luminoth/utils/image_vis.py

```diff
@@ -90,29 +115,34 @@ def get_image_summaries(summaries_fn, pred_dict, extra_tag=None):
                 tag = os.path.join(fn_name, ','.join(
                     '{}={}'.format(k, v) for k, v in argument.items()
                 ))
-
+            if 'gt_bboxes' in inspect.getargspec(globals()[fn_name]).args:
```

Comment:
Why not just add gt_bboxes to pred_dict to avoid having to do this?

Reply:
I didn't think that wold be very clear but I can change it